### PR TITLE
Add MSW XVI and Novation Launchpad Pro MK3

### PR DESCRIPTION
**Pull request by naim**

### CODE REVIEW

File: devices.ttl [hunk 1]
```diff
@@ -697,3 +697,18 @@ _:polyend-tracker device:typeB true .
...
+_:michigan-synth-works-xvi device:model "XVI" .
+_:michigan-synth-works-xvi device:outputs 1 .
+_:michigan-synth-works-xvi device:typeB true .
+_:michigan-synth-works-xvi device:details "XVI Desktop USB 16 Channel Fader Bank with CV, I2C, and MIDI" .
+_:michigan-synth-works-xvi device:uri "https://michigansynthworks.com/products/xvi-desktop-usb-16-channel-fader-bank-with-cv-i2c-and-midi" .
+
+_:novation-launchpad-pro-mk3 device:make "Novation" .
+_:novation-launchpad-pro-mk3 device:model "Launchpad Pro (mk3)" .
+_:novation-launchpad-pro-mk3 device:inputs 1 .
+_:novation-launchpad-pro-mk3 device:outputs 1 .
```
Comment (audionerd) [inline devices.ttl:710]: From the description/photos, it looks like 2 outs which can also be thru.

File: devices.ttl [hunk 2]
```diff
@@ -697,3 +697,18 @@ _:polyend-tracker device:typeB true .
 _:polyend-tracker device:uri "https://polyend.com/tracker" .
 _:polyend-tracker device:details "via https://help.polyend.com/639766-Is-the-Polyend-Tracker-MIDI-compatible" .
 
+_:michigan-synth-works-xvi device:make "Michigan Synth Works" .
+_:michigan-synth-works-xvi device:model "XVI" .
+_:michigan-synth-works-xvi device:outputs 1 .
+_:michigan-synth-works-xvi device:typeB true .
```
Comment (audionerd) [inline devices.ttl:703]: The description on the MiSW site says:

> As of Serial#4161, all devices have TRS MIDI A

Maybe we should record the XVI as `device:typeA` (with a `device:notes` fields saying that models before Serial#4161 were Type B). Or does the XVI accept both types?

Comment (naim) [reply devices.ttl:703]: Ah, good catch. The top-level description still just says B. It doesn't accept both that I'm aware of. I'll update it to `typeA` and add an appropriate note.

**Comment by audionerd** [comment]
Merged! Thanks for your help!

**Pull request merged**